Dockerfile

```diff
@@ -1,12 +1,16 @@
 # Install pytest python library as well as add all files in current directory
-FROM python:3 AS base
+FROM python:3.7 AS base
 WORKDIR /usr/src/app
 RUN apt-get update \
     && apt-get install -y enchant \
     && rm -rf /var/lib/apt/lists/*
-RUN pip install --upgrade pip
+
 RUN pip install coveralls
-ADD . .
+ADD requirements.txt ./
 RUN pip install --no-cache-dir -r requirements.txt
+
+ADD . .
```

Comment:
Don't ADD but COPY, and COPY specifically the files/directories you need.

Reply:
:+1:

Reply:
I do think we need to add all files though